lib/order.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'menu'
+
+class Order
+
+  attr_reader :list, :menu, :order, :total_cost
+
+  def initialize(menu = Menu.new)
+    @list = []
+    @menu = menu
+  end
+
+  def read_menu
+    @menu.display_menu
+  end
+
+  def choose(item)
+   @choice = @menu.set_menu.select { |key, value| item.include? key }
```

Comment:
Lint/UnusedBlockArgument: Unused block argument - `value`. If it's necessary, use `_` or `_value` as an argument name to indicate that it won't be used. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)